libraries/wallet/wallet.cpp

```diff
@@ -2194,6 +2198,10 @@ fc::ecc::private_key wallet_api::derive_private_key(const std::string& prefix_st
             SET_PROP(max_referral_interest_rate);
             SET_PROP(max_referral_term_sec);
             SET_PROP(max_referral_break_fee);
+            SET_PROP(comments_window);
```

Comment:
Here should be different types of chain_properties.
You will merge changes in other commit?

Reply:
@afalaleev Yes, will fix conflicts after all work.